features/environment.py

```diff
@@ -22,25 +22,34 @@ class UAClientBehaveConfig:
         A string with an image name that should be used instead of building a
         fresh image for this test run.  If specified, image_clean will be set
         to False.
+    :param preserve_instance:
```

Comment:
We launch many containers during test runs, not just one; should the option be plural?